Sources/LLVM/StructType.swift

```diff
@@ -35,10 +35,16 @@ public struct StructType: IRType {
   /// - parameter elementTypes: A list of types of members of this structure.
   /// - parameter isPacked: Whether or not this structure is 1-byte aligned with
   ///   no packing between fields.  Defaults to `false`.
-  public init(elementTypes: [IRType], isPacked: Bool = false) {
+  /// - parameter context: The context to create this type in
+  /// - SeeAlso: http://llvm.org/docs/ProgrammersManual.html#achieving-isolation-with-llvmcontext
+  public init(elementTypes: [IRType], isPacked: Bool = false, in context: Context? = nil) {
```

Comment:
This one is still optional

Reply:
Just call the `...InContext` one here.  It's what happens eventually on the else branch anyway.